### PR TITLE
Fix double onFrameChange events with AnimatedSprite

### PR DESCRIPTION
**Pull request by TotooriaHyperion**
##### Description of change

if it's stoped naturally, then reset frame to the last frame and don't trigger one more onFrameChange
##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: packages/sprite-animated/test/AnimatedSprite.js [hunk 1]
```diff
@@ -95,4 +95,214 @@ describe('PIXI.AnimatedSprite', function ()
...
+
+        it('should fire every frame(except current) during one play', function (done)
+        {
+            const frameIds = [];
+
+            this.sprite.gotoAndStop(0);
+            this.sprite.animationSpeed = 1;
+            this.sprite.onComplete = () =>
+            {
+                done();
```
Comment (bigtimebuddy) [inline packages/sprite-animated/test/AnimatedSprite.js]: Might be better to call done after all the other expect statements?

File: packages/sprite-animated/test/AnimatedSprite.js [hunk 2]
```diff
@@ -95,4 +95,214 @@ describe('PIXI.AnimatedSprite', function ()
...
+
+        it('should fire every frame(except current) during one play - reverse', function (done)
+        {
+            const frameIds = [];
+
+            this.sprite.gotoAndStop(2);
+            this.sprite.animationSpeed = -1;
+            this.sprite.onComplete = () =>
+            {
+                done();
```
Comment (bigtimebuddy) [inline packages/sprite-animated/test/AnimatedSprite.js]: Same here. Add done to the end of this callback.

File: packages/sprite-animated/test/AnimatedSprite.js (hunk 1 above)
Comment (TotooriaHyperion) [reply packages/sprite-animated/test/AnimatedSprite.js]: yes, my stupid mistake.

**Comment by bigtimebuddy** [comment]
Thank you!

**Comment by ivanpopelyshev** [comment]
Guys, this thing fails on windows :(

**Comment by bigtimebuddy** [comment]
Tests or runtime fails @ivanpopelyshev? Can you be more specific?

**Pull request merged**